receiver/hostmetricsreceiver/internal/scraper/processscraper/documentation.md

```diff
@@ -12,6 +12,9 @@ These are the metrics available for this scraper.
 | **process.disk.io** | Disk bytes transferred. | By | Sum(Int) | <ul> <li>direction</li> </ul> |
 | **process.disk.io.read** | Disk bytes read. | By | Sum(Int) | <ul> </ul> |
 | **process.disk.io.write** | Disk bytes written. | By | Sum(Int) | <ul> </ul> |
+| **process.disk.operations** | Number of disk operations performed by the process. | {operations} | Sum(Int) | <ul> <li>direction</li> </ul> |
```

Comment:
metric name cannot be uses as a namespace of other metrics according to the spec. please rename it. maybe add `.count` to the end?

Reply:
The additional metrics `process.disk.operations.read` and `process.disk.operations.write` were only to accommodate the `emitMetricsWithDirectionAttribute` feature flag. This feature flag is now removed, so I'll remove these metrics too.